src/components/DataSubmissions/ValidationControls.tsx

```diff
@@ -159,12 +160,9 @@ const ValidationControls: FC = () => {
     if (dataSubmission.intention === "Delete" || dataSubmission.dataType === "Metadata Only") {
       return false;
     }
-    if (collaborator && collaborator.permission !== "Can Edit") {
-      return false;
-    }
 
     return dataSubmission?.fileValidationStatus !== null;
-  }, [user?.role, dataSubmission?.fileValidationStatus, dataSubmission?.status, collaborator]);
+  }, [user?.role, dataSubmission]);
```

Comment:
We may want to change `user?.role` to just `user` now?

```suggestion
  }, [user, dataSubmission]);
```

Reply:
Thanks, yes I updated it for `canValidateMetadata` as well in [1f7d50a](https://github.com/CBIIT/crdc-datahub-ui/pull/573/commits/1f7d50a726bae024531f20dfc295a24e709f94c7).